pkg/venafi/tpp/connector.go

```diff
@@ -270,6 +277,14 @@ func (c *Connector) RevokeAccessToken(auth *endpoint.Authentication) (err error)
 
 func processAuthData(c *Connector, url urlResource, data interface{}) (resp interface{}, err error) {
 
+	//isReachable, err := c.isAuthServerReachable()
```

Comment:
Hey @rvelaVenafi . Help me understand, what is the reason to have this commented?

I can provide why this method is needed even if Ping() is present. If you take a look on the URL of Ping() and this method you will see that they are using two different IIS application pools. This means that API clients will need bring them up in order to avoid timeouts. Let me know if you need additional details.